plugins/modules/zos_blockinfile.py

```diff
@@ -38,6 +38,8 @@
       - The location can be a UNIX System Services (USS) file,
         PS (sequential data set), member of a PDS or PDSE, PDS, PDSE.
       - The USS file must be an absolute pathname.
+      - Generation data set (GDS) relative name of generation already
+        created SOME.CREATION(-1).
```

Comment:
When adding the example is better to change the style of it and indicate is an example.
```suggestion
        created.  C(e.g. SOME.CREATION(-1).)
```